template/mkdocs.yml

```diff
@@ -6,4 +6,4 @@ pages:
 site_name: Component Name Goes Here
 site_description: 'Component Description goes here'
 repo_url: 'https://github.com/{org}/{repo}'
```

Comment:
Is the placeholder `{org}` really needed? The description of this repository is clearly:

> This is a repository containing information and tools for contributors to and maintainers of Zend Framework.

https://github.com/zendframework/maintainers

Reply:
@webimpress 
Okay, thanks for the info!